earthpy/clip.py

```diff
@@ -30,13 +29,11 @@ def _clip_points(shp, clip_obj):
 
 
 def _clip_line_poly(shp, clip_obj):
-    """A function to clip line and polygon data using geopandas.
+    """Clip line and polygon data using geopandas.
```

Comment:
How about:
`Clip line and polygon geometries to the clip_obj GeoDataFrame geometry.`